types/src/shared/deployment.ts

```diff
@@ -0,0 +1,63 @@
+import * as child_process from "child_process";
+
+import { LoggerInterface } from "./logger";
+
+export async function sendDeploymentMessage({
```

Comment:
Why do we need this ? We already have deployments on slack no ?

Reply:
Ah my bad, it's for initdb.

Reply:
Agree that the name is incorrect.